components/uniset/src/uniset.rs

```diff
@@ -13,7 +13,8 @@ const BMP_MAX: u32 = 0xFFFF;
 ///
 /// Provides exposure to membership functions and constructors from serialized [`UnicodeSets`](UnicodeSet)
 /// and predefined ranges.
-#[derive(Debug, PartialEq, Hash, Eq)]
+#[derive(Debug, PartialEq, Hash, Eq, Clone)]
+#[cfg_attr(feature = "provider_serde", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
Nit: As described [in the new features documentation](https://unicode-org.github.io/icu4x-docs/doc/icu/index.html#features), use feature "serde" for library types like this one.